content/id/docs/reference/glossary/rbac.md

```diff
@@ -0,0 +1,18 @@
+---
+title: RBAC (Role-Based Access Control)
```

Comment:
Can we add Indonesian translation in the title? “Akses kontrol berbasis Rol”?

Reply:
I hesitated to put "Akses kontrol berbasis Rol" there for title, because then it feels wrong to use the RBAC abbreviation. Or should we change it to "RBAC" only, then in the explanation we can put:

RBAC (_Role-Based Access Control_) atau akses kontrol berbasis rol digunakan untuk mengelola keputusan otorisasi dan memungkinkan admin untuk mengonfigurasi kebijakan akses secara dinamis melalui API Kubernetes.

Reply:
How to put "RBAC (_Role-Based Access Control_) or Akses Kontrol Berbasis Rol" in the title? too long?

Reply:
I'd expect to see **Kontrol Akses Berbasis Rol** in the title and **Role-Based Access Control** listed under `aka:`

Reply:
Nice to know the usage of `aka` field there. Thanks @sftim.